cwms-data-api/src/test/java/fixtures/CwmsDataApiSetupCallback.java

```diff
@@ -87,6 +89,30 @@ public void beforeAll(ExtensionContext context) throws Exception {
                        .jsonConfig(
                             JsonConfig.jsonConfig()
                                       .numberReturnType(JsonPathConfig.NumberReturnType.DOUBLE));
+            healthCheck();
+        }
+    }
+
+    private static void healthCheck() throws InterruptedException {
+        int attempts = 0;
+        int maxAttempts = 15;
+        for (; attempts < maxAttempts; attempts++) {
+            try {
+                given()
+                .when()
+                    .get("/offices/")
```

Comment:
if you put a specific office at the end of this, like `/offices/SPK` you can check for a 200 OK and the response should also be slightly faster.

Reply:
great point, updated